spec/features/form_sections/teacher_training_data/placements/deleting_trainee_placements_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Delete a placement" do
+  after do
+    FormStore.clear_all(@trainee.id)
+  end
+
+  scenario "Delete a placement from an existing trainee" do
+    given_i_am_authenticated
+    and_a_trainee_exists_with_a_placement
+
+    when_i_navigate_to_the_delete_placement_form
+    then_i_see_the_not_found_page
```

Comment:
That should be it's own test

Similar to 
https://github.com/DFE-Digital/register-trainee-teachers/blob/35538b37ae4608f9eb08b6c45df24bb381e84e8d/spec/features/form_sections/teacher_training_data/placements/confirm_spec.rb#L28-L34